api/controllers/ApplicationTypeController.js

```diff
@@ -61,179 +50,192 @@ module.exports = {
      */
     serviceSelectorPage(req, res) {
         // clear down eligibility checker selected documents
-        req.session.search_history =[];
+        req.session.search_history = [];
         //reset the selected documents
         req.session.selectedDocuments = {
             totalDocCount: 0,
-            documents: []
+            documents: [],
         };
+        req.session.appId = false; // reset the appId so a new session is used
+        // set initial submit status to false, meaning it application has not yet been submitted
+        req.session.appSubmittedStatus = false;
 
-      if (req.query.newSession){
```

Comment:
Yes when the user logs out it redirects them back to the /select-service page. I think that change was done a while ago so I can't remember exactly why it was made, the change to redirect to the service page. But if you think it's best to keep this code just in case, I'm happy to put it back.